roles/os_hardening/tasks/cron.yml

```diff
@@ -0,0 +1,84 @@
+---
+# Granting write access to this directory for non-privileged users could provide
+# them the means for gaining unauthorized elevated privileges.
+# Granting read access to this directory could give an unprivileged user insight
+# in how to gain elevated privileges or circumvent auditing controls.
+# CIS 5.1.2 - CIS 5.1.7
+#
+- name: Check if /etc/crontab exists
+  stat:
+    path: /etc/crontab
+  register: osh_crontab
```

Comment:
What's up with `osh`? Maybe `file_crontab` or `dir_cron_hourly` would be better?

Reply:
I figured that you would comment on this one ;-) I think it would be a good idea if all the variable names in the role or collection were unique to minimize the risk that they will collide. I figured osh is OS Hardening. This way if I have a play that adds a task in crontab that uses a variable named crontab, it won't collide even if they are in the same playbook.

I can remove the "osh" if you want me to. This was just a way for me to bring this up for discussion.